io/net/ethtool_test.py

```diff
@@ -193,10 +193,24 @@ def test_ethtool(self):
                                         else:
                                             self.fail("%s %s" % (
                                                 self.args, result.stderr_text))
-                    cmd = "ethtool %s %s %s %s %s %s %s %s %s %s" % (
-                        self.args, self.interface, self.param[0], value[0],
-                        self.param[1], value[1], self.param[2], value[2],
-                        self.param[3], value[3])
+                    #cmd = "ethtool %s %s %s %s %s %s %s %s %s %s" % (
+                    cmd = (
+                        f"ethtool {self.args} {self.interface} {self.param[0]} {value[0]}"
+                        f"{self.param[1]} {value[1]} {self.param[2]} {value[2]}"
+                        f"{self.param[3]} {value[3]}"
+                        )
+                    ret = process.run(cmd, shell=True, verbose=True,
+                                      ignore_status=True)
+                    if ret.exit_status != 0:
+                        #self.fail("%s %s" % (self.args, ret.stderr_text))
+                        self.fail(f"{self.args} {ret.stderr_text}")
+                    # Set queue values to original values
+                    #cmd = "ethtool %s %s %s %s %s %s %s %s %s %s" % (
```

Comment:
@vaishnavibhat Thanks for incorporating the changes, can you please remove the commented lines?